scripts/generate_manifest.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+# ./scripts/generate_manifest.sh <DOCKERFILE> <MANIFEST_PATH>
+# Example:
+# ./scripts/generate_manifest.sh webapp/Dockerfile manifest_webapp.txt
+# cat manifest_webapp.txt
+
+DOCKERFILE=$1
+MANIFEST_PATH=$2
+DOCKER_CONTEXT=$3
+
+echo "Dockerfile:    ${DOCKERFILE}"
+echo "Manifest path: ${MANIFEST_PATH}"
+
+# Write base image name.
+echo "[base_image]" > ${MANIFEST_PATH}
+cat ${DOCKERFILE} | grep "FROM" | sed -e "s/FROM //" >> ${MANIFEST_PATH}
+
+
+# Build image and write out included packages (rpm, pip).
+echo "Building image to inspect..."
+docker build -t mf_image -f ${DOCKERFILE} .
+
+## Write content of /etc/system-release-cpe
+echo "[system-release-cpe]" >> ${MANIFEST_PATH}
+cat /etc/system-release-cpe
```

Comment:
This line probably shouldn't be here as it prints out CPE of the host machine which is not necessary.